src/components/Comment/index.spec.tsx

```diff
@@ -200,6 +200,22 @@ describe(__filename, () => {
     expect(button).toHaveText('Saving…');
   });
 
+  it('can submit an empty comment', () => {
```

Comment:
Why do we need to be able to submit an empty comment? That seems like something that should not be allowed? What's the value of an empty comment?

Reply:
Nothing, no value. I added this test after you suggested adding coverage of [the new `null` check](https://github.com/mozilla/addons-code-manager/pull/1083/files#diff-6cbfe0c60e904609a838a8d33353288fR200). I should delete the test, though, because it doesn't really cover that `null` check.

Reply:
I couldn't really think of a way to regression test that `null` conversion. I suggest that we just be more vigilant in code reviews of type safety problems related to https://github.com/microsoft/TypeScript/issues/13778 . I did see https://github.com/danielnixon/total-functions/ as a potential workaround but it might make some JavaScript idioms more complicated, I'm not sure.